README.rst

```diff
@@ -1,207 +1,192 @@
+Tyler, Nathaniel Pryce, Dave Wallace, John Popplewell, Michael Urman,
+Andrew Straw, Michael Hudson, Ole Martin Bjoerndalen, Herve Cauwelier,
+James Mazer, Lalo Martins, Timothy Stranex, Chad Lester, Matthias
+Spiller, Bo Jangeborg, Dmitry Borisov, Campbell Barton, Diego Essaya,
+Eyal Lotem, Regis Desgroppes, Emmanuel Hainry, Randy Kaelber
+Matthew L Daniel, Nirav Patel, Forrest Voight, Charlie Nolan,
+Frankie Robertson, John Krukoff, Lorenz Quack, Nick Irvine,
+Michael George, Saul Spatz, Thomas Ibbotson, Tom Rothamel, Evan Kroske,
+Cambell Barton.
+
+And our bug hunters above and beyond: Angus, Guillaume Proux, Frank
+Raiser, Austin Henry, Kaweh Kazemi, Arturo Aldama, Mike Mulcheck,
+Michael Benfield, David Lau
+
+There's many more folks out there who've submitted helpful ideas, kept
+this project going, and basically made our life easier.  Thanks!
+
+Many thank you's for people making documentation comments, and adding to the
+pygame.org wiki.
+
+Also many thanks for people creating games and putting them on the
+pygame.org website for others to learn from and enjoy.
+
+Lots of thanks to James Paige for hosting the pygame bugzilla.
+
+Also a big thanks to Roger Dingledine and the crew at SEUL.ORG for our
+excellent hosting.
 
 Dependencies
-============
-
-     Pygame is obviously strongly dependent on SDL and Python. It also
-     links to and embeds several other smaller libraries. The font
-     module relies on SDL_tff, which is dependent on freetype. The mixer
-     (and mixer.music) modules depend on SDL_mixer. The image module
-     depends on SDL_image, which also can use libjpeg and libpng. The
-     transform module has an embedded version of SDL_rotozoom for its
-     own rotozoom function. The surfarray module requires the Python
-     NumPy package for its multidimensional numeric arrays.
-     Dependency versions:
-
-     * Python >= 2.7 or PyPy >= 6.0.0 (and pypy3)
-     * SDL >= 1.2.15
-     * SDL_mixer >= 1.2.13
-     * SDL_image >= 1.2.12
-     * SDL_tff >= 2.0.11
-     * SDL_gfx (optional, vendored in)
-     * NumPy >= 1.6.2 (optional)
+------------
+
+Pygame is obviously strongly dependent on SDL and Python.  It also
+links to and embeds several other smaller libraries.  The font
+module relies on SDL_tff, which is dependent on freetype.  The mixer
+(and mixer.music) modules depend on SDL_mixer.  The image module
+depends on SDL_image, which also can use libjpeg and libpng.  The
+transform module has an embedded version of SDL_rotozoom for its
+own rotozoom function.  The surfarray module requires the Python
+NumPy package for its multidimensional numeric arrays.
+Dependency versions:
+
+* CPython >= 2.7 or PyPy >= 6.0.0 (and pypy3)
+* SDL >= 1.2.15
+* SDL_mixer >= 1.2.13
+* SDL_image >= 1.2.12
+* SDL_tff >= 2.0.11
+* SDL_gfx (optional, vendored in)
+* NumPy >= 1.6.2 (optional)
 
 
 Contribute
-==========
+----------
 
-     https://www.pygame.org/contribute.html
```

Comment:
Maybe numpy could be (optional, but recommended).

We also have some other dependencies that could be added. Like portmidi, and freetype. 

The dependencies have dependencies (like SDL_image uses libpng, libjpeg and others)... I wonder if they are worth listing here(as an indented sub list?).

Another improvement here would be to make these links.

Reply:
I suppose that we can list them in the linked wiki page.  I don't really have an opinion on this, all I did was reformatting so that it looks better rendered :smile: